### PR TITLE
DEV-772  create open stack error page

### PR DESCRIPTION
**Pull request by Cassiepang**

### CODE REVIEW

File: src/main/java/sg/ncl/MainController.java [hunk 1]
```diff
@@ -179,6 +179,12 @@ public String jointeam() {
         return "jointeam";
     }
 
+
+    @RequestMapping("/Error_OpenStack")
+    public String Error_OpenStack() {
```
Comment (ChrisZhong) [inline src/main/java/sg/ncl/MainController.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

File: src/main/java/sg/ncl/MainController.java [hunk 2]
```diff
@@ -179,6 +179,12 @@ public String jointeam() {
...
+        return "Error_OpenStack";
```
Comment (ChrisZhong) [inline src/main/java/sg/ncl/MainController.java]: MINOR Remove this method and declare a constant for this value. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)

**Comment by ChrisZhong** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

File: src/main/java/sg/ncl/MainController.java (hunk 1 above)
Comment (ZHANGChunwang) [inline src/main/java/sg/ncl/MainController.java]: Pls change "/Error_OpenStack" to "/error_openstack", change Error_OpenStack() to errorOpenStack()

File: src/main/java/sg/ncl/MainController.java (hunk 2 above)
Comment (ZHANGChunwang) [inline src/main/java/sg/ncl/MainController.java]: Pls change "Error_OpenStack" to "error_openstack"

File: src/main/resources/templates/Error_OpenStack.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	<p>&nbsp;</p>
+
+	<section id="joinUs" class="join-us">
+		<div class="container" >
+			<div class="row">
+				<div class="box effect5">
+					<div class="heading text-center col-sm-12">
+						<h4>Currently, we are in the progress of integrating OpenStack with our systems.<p>&nbsp;</p>
+
+							If you are interested to find out more, email us at <strong>support@ncl.sg </strong></h4>
```
Comment (ZHANGChunwang) [inline src/main/resources/templates/Error_OpenStack.html]: Is there a way to declare "support@ncl.sg" as a global variable?

File: src/main/resources/templates/fragments/dashboard_navigation.html [hunk 4]
```diff
@@ -53,7 +53,7 @@
 						<li><a href="TestbedInformation.html" th:href="@{/TestbedInformation}">Testbed Info</a></li>
 					</ul>
 				</li>
-				<li><a href="" th:href="@{/OpenStack}"><navsmall>Switch to OpenStack</navsmall></a></li>
+				<li><a href="Error_OpenStack.html" th:href="@{/Error_OpenStack}"><navsmall>Switch to OpenStack</navsmall></a></li>
```
Comment (ZHANGChunwang) [inline src/main/resources/templates/fragments/dashboard_navigation.html]: Error_OpenStack.html --> error_openstack.html, 
/Error_OpenStack --> /error_openstack

**Pull request merged**